docs/swagger.yaml

```diff
@@ -2144,6 +2300,13 @@ components:
           $ref: '#/components/schemas/MerchantInfoPayment'
         transaction:
           $ref: '#/components/schemas/CancelTransaction'
+    AuthroizePaymentRequest:
+      type: object
+      properties:
+        merchantInfo:
+          $ref: '#/components/schemas/MerchantInfoPayment'
+        transaction:
+          $ref: '#/components/schemas/CancelTransaction'
```

Comment:
Maybe this needs renaming now, how about `SimpleTransaction` and `ValueTransaction` for transaction objects with and without a "amount" property?

Reply:
Let's take that in a separate PR, not sure if I agree.